.github/workflows/prepare-tanssi-relay-binary.yml

```diff
@@ -0,0 +1,114 @@
+name: Prepare Optimized Dancelight Binary Draft (internal only)
+
+# The code (like generate-release-body) will be taken from the tag version, not master
+on:
+  push:
```

Comment:
you can remove this now

Reply:
remove all push triggers in https://github.com/moondance-labs/tanssi/pull/696/commits/f0091a732e74dbefd5ed161aee38297259b3031e